pkg/s3select/select-datatypes.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Minio Cloud Storage, (C) 2015, 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
We can just say this as datatypes.go not `select-datatypes.go`

Reply:
Done.